src/test/java/test/LoginPageTest.java

```diff
@@ -0,0 +1,61 @@
+package test;
+
+import base.BaseTest;
+import config.ConfigReader;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+
+import static org.testng.Assert.*;
+
+public class LoginPageTest extends BaseTest {
+    private String username;
+    private String password;
+    private LoginPage loginPage;
+
+    @BeforeMethod
+    public void setUp() {
+        initializeDriver();
+        String baseUrl = ConfigReader.get("baseUrl");
+        username = ConfigReader.get("username");
+        password = ConfigReader.get("password");
+        driver.get(baseUrl);
+        loginPage = new LoginPage(driver);
+    }
+
+    @Test
+    public void testValidLogin() {
+        loginPage.login(username, password);
```

Comment:
should not 1 step login, please define step-by-step: input username, input password, click login button. we can leave that login() to use as pre-condition in @BeforeMethod or @BeforeClass
e.g:
```suggestion
        loginPage.inputUsername(username);
        loginPage.inputPassword(password);
        loginPage.clickLoginBtn();
}
```